testsuite/tests/lib-unix/isatty/isatty.ml

```diff
@@ -0,0 +1,7 @@
+Printf.printf
+  "Unix.isatty Unix.stdin = %b\n\
+   Unix.isatty Unix.stdout = %b\n\
+   Unix.isatty Unix.stderr = %b"
+  (Unix.isatty Unix.stdin)
+  (Unix.isatty Unix.stdout)
+  (Unix.isatty Unix.stderr)
```

Comment:
I would feel better if at least one test of `Unix.isatty` would return `true`.  For example, what about opening `/dev/tty` or `CON` (whichever succeeds) and test that?